### ISSUE TITLE
Incorect API doc generation (form parser issues?)

### ISSUE THREAD

**Issue by dragosprotung** [nelmio/NelmioApiDocBundle#785]
Having a form defined as:

```php
public function buildForm(FormBuilderInterface $builder, array $options)
{
   $builder
      ->add('sort', SortType::class)
      ->add('filter', FilterType::class);
}
```
attached to a controller doc block will render the filter (or any next fields) with the same values as the sort (the first defined).

After downgrading to version 2.10.3 the output is as expected.

**Comment by dragosprotung** [comment]
Seems to be the same problem like in #771

**Comment by dragosprotung** [comment]
Fixed in  PR #799
